stripe/http_client.py

```diff
@@ -282,22 +282,22 @@ def request(self, method, url, headers, post_data=None):
             curl.perform()
         except pycurl.error as e:
             self._handle_request_error(e)
-        rbody = s.getvalue()
+        rbody = b.getvalue().decode('utf-8')
         rcode = curl.getinfo(pycurl.RESPONSE_CODE)
 
-        return rbody, rcode, self.parse_headers(rheaders.getvalue())
+        return rbody, rcode, self.parse_headers(rheaders.getvalue().decode('utf-8'))
 
     def _handle_request_error(self, e):
-        if e[0] in [pycurl.E_COULDNT_CONNECT,
-                    pycurl.E_COULDNT_RESOLVE_HOST,
-                    pycurl.E_OPERATION_TIMEOUTED]:
+        if e.args[0] in [pycurl.E_COULDNT_CONNECT,
+                         pycurl.E_COULDNT_RESOLVE_HOST,
+                         pycurl.E_OPERATION_TIMEOUTED]:
```

Comment:
Just wanted to flag this as it looks like GitHub's detecting some whitespace changes. Was that on purpose?